vue.md

```diff
@@ -0,0 +1,51 @@
+# Vue
+
+**Prerequisite knowledge**: js, es6, single-page application, framework, npm
+
+## Intro
+Vue.js (commonly referred to as Vue; pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces and single-page applications. Vue.js features an incrementally adoptable architecture that focuses on declarative rendering and component composition. Advanced features required for complex applications such as routing, state management and build tooling are offered via officially maintained supporting libraries and packages. [[Source](https://en.wikipedia.org/wiki/Vue.js)]
+
+## Classification / acceptance questions
+
+### Apprentice aka _Hello World_
+* I can instantiate Vue object
+* I can create a component
+* I understand Vue component lifecycle
+* I can use Vue template syntax to render component HTML
+
+### Intermediate aka _My first SPA_
+**Prerequisite knowledge**: sass
+* I can use conditional rendering (`v-if` vs `v-show`)
```

Comment:
I would mention different ways of registering components:
- global components
- import component file inside different component

Reply:
> import component file inside different component
IMO this is a default way to import components. I'll mention about global components somewhere.